src/pykeen/triples/triples_factory.py

```diff
@@ -125,6 +130,10 @@ def _map_triples_elements_to_ids(
     relation_to_id: RelationMapping,
 ) -> MappedTriples:
     """Map entities and relations to pre-defined ids."""
+    if triples.size < 1:
```

Comment:
can't this just be a zero check? Are negative sizes possible? 😛

Reply:
9d34aa5 (although functionally the same) :wink: